exe/kubernetes-deploy

```diff
@@ -12,7 +12,8 @@ bindings = {}
 verbose_log_prefix = false
 
 ARGV.options do |opts|
-  opts.on("--bindings=BINDINGS", "Expose additional variables to ERB templates (format: k1=v1,k2=v2 or JSON)") do |binds|
+  opts.on("--bindings=BINDINGS", "Expose additional variables to ERB templates " \
```

Comment:
Same comment as on your other PR (fine to fix this, but we should 🎩  every time we change this file since it is untested)

Reply:
🎩 tested.